eventstore/dynamo/dynamo.go

```diff
@@ -0,0 +1,193 @@
+package dynamo
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/hallgren/eventsourcing/core"
+)
+
+type DynamoEventStore struct {
```

Comment:
Could you add some documentation on how to use this?
I thinking mainly in terms of DynamoDB, i.e. which PK and SK is assumed etc.